css/types/color.json

```diff
@@ -1011,9 +1011,16 @@
               "web-features:oklab"
             ],
             "support": {
-              "chrome": {
-                "version_added": "111"
-              },
+              "chrome": [
+                {
+                  "version_added": "122"
+                },
+                {
+                  "version_added": "111",
+                  "partial_implementation": true,
+                  "notes": "`l` channel values incorrectly resolve to numbers between 0-100 rather than 0-1. As a result, channel value calculations require `l` values to be specified as percentage numbers without units (e.g. 20 for 0.2). See [bug 40940488](https://crbug.com/40940488)."
+                }
```

Comment:
This change should go in the `relative_syntax` subfeature below (note that it is only supported in Chrome 119+).